app.rb

```diff
@@ -101,4 +102,58 @@ def create_person
       nil
     end
   end
+
+  def save_people_data
+    File.write('people.json', JSON.pretty_generate(
+                                @people.map do |person|
+                                  if person.instance_of?(Teacher)
+                                    { name: person.name, age: person.age, parent_permission: person.parent_permission,
+                                      specialization: person.specialization }
+                                  else
+                                    { name: person.name, age: person.age, parent_permission: person.parent_permission }
+                                  end
+                                end
+                              ))
```

Comment:
- [ ] when I tried to save people it doesn't please make sure when a student or teacher created should be saved as well in order to complete the requirements